pkg/kubernetes/policy_report_client.go

```diff
@@ -40,6 +40,9 @@ func (k *k8sPolicyReportClient) Stop() {
 func (k *k8sPolicyReportClient) Sync(stopper chan struct{}) error {
 	factory := metadatainformer.NewSharedInformerFactory(k.metaClient, 15*time.Minute)
 
+	// tcInformer := tcinformer.NewSharedInformerFactory(tcv1alpha1.New(&rest.RESTClient{}), time.Second)
```

Comment:
I assume you wanted to try things out, don't forget to clean everything up again.